Week 1.py

```diff
@@ -0,0 +1,109 @@
+
+# coding: utf-8
+
+# In[27]:
+
+
+def reverse_integer(x):
```

Comment:
- X, x
- for i in range(len(L)), thay a = length
- Look up: descending syntax. range(10, 1, -1)
Tests:
x = 0: print -0.
-120
-12000